packages/playground-examples/copy/pt/Typescript/Primitives/Literals.ts

```diff
@@ -0,0 +1,64 @@
+// TypeScript tem alguns casos engraçados para literais em seu código
+
+// Grande parte desse suporte é abordado em widening
+// and narrowing ( exemplo:type-widening-narrowing ) e vale
```

Comment:
```suggestion
// and narrowing ( example:type-widening-narrowing ) e vale
```
This is a hidden hyperlink, and should use the `example:*` syntax

Reply:
Oh sure, I'll fix that.